formfyxer/pdf_wrangling.py

```diff
@@ -80,8 +97,11 @@ def __init__(
             config: a dictionary containing any keyword argument to the reportlab field functions,
                 which will vary depending on what type of field this is. See section 4.7 of the
                 [reportlab User Guide](https://www.reportlab.com/docs/reportlab-userguide.pdf)
+            program_name: the programmatic name of the field. Not the tooltip, but `users1_name__0`
```

Comment:
can we just stick with `field_name` here? consistent with the usage we have in other packages. "Programmatic" doesn't mean anything to me, and shouldn't be shortened to program.

Reply:
I agreed, but yeah, for some reason decided that was where I drew the line for this PR's refactoring. I think I got the names from report lab (which this class acted as a interface for).

I can make a quick PR after this is merged.